dojo/tools/bundler_audit/parser.py

```diff
@@ -91,8 +93,8 @@ def get_findings(self, filename, test):
                     component_name=gem_name,
                     component_version=gem_version,
```

Comment:
could you please use advisory_id to fill this into Finding.cve? 
https://github.com/DefectDojo/django-DefectDojo/blob/c5d896b2a606985f41669388962e47409f4866ba/dojo/models.py#L2206
I guess Finding.cve also can handle GHSA as far as I know.

Reply:
https://github.com/DefectDojo/django-DefectDojo/commit/626defd50904b720429a8dd37f0b7891242ee014